### PR TITLE
Fixed broken links

### PR DESCRIPTION
**Pull request by RenuPatelGoogle**
Fixed broken links for `Keras API specifications`, `Using Model.fit`, `Sequential`, `Functional` , `subclassed` links in section 'Use tf.distribute.Strategy with Keras Model.fit'  and  for `with custom training loops` link in section 'Use tf.distribute.Strategy with custom training loops'. I have provided the .ipynb file path as mentioned in the broken link.. Please suggest in case any changes are required.

### CODE REVIEW

File: site/en/guide/distributed_training.ipynb [hunk 1]
```diff
@@ -503,14 +503,14 @@
       "source": [
         "## Use tf.distribute.Strategy with Keras Model.fit\n",
         "\n",
-        "`tf.distribute.Strategy` is integrated into `tf.keras`, which is TensorFlow's implementation of the [Keras API specification](https://keras.io). `tf.keras` is a high-level API to build and train models. By integrating into the `tf.keras` backend, it's seamless for you to distribute your training written in the Keras training framework [using Model.fit](/keras/customizing_what_happens_in_fit.ipynb).\n",
+        "`tf.distribute.Strategy` is integrated into `tf.keras`, which is TensorFlow's implementation of the [Keras API specification](https://keras.io/api/). `tf.keras` is a high-level API to build and train models. By integrating into the `tf.keras` backend, it's seamless for you to distribute your training written in the Keras training framework [using Model.fit](keras-io/guides/ipynb/customizing_what_happens_in_fit.ipynb).\n",
```
Comment (MarkDaoust) [inline site/en/guide/distributed_training.ipynb:506]: keras-io/ doesn't work either.

You want
```suggestion
        "`tf.distribute.Strategy` is integrated into `tf.keras`, which is TensorFlow's implementation of the [Keras API specification](https://keras.io/api/). `tf.keras` is a high-level API to build and train models. By integrating into the `tf.keras` backend, it's seamless for you to distribute your training written in the Keras training framework [using Model.fit](https://www.tensorflow.org/guide/keras/customizing_what_happens_in_fit).\n",
```

Comment (MarkDaoust) [reply site/en/guide/distributed_training.ipynb:506]: Same for all the other links.

Comment (8bitmp3) [inline site/en/guide/distributed_training.ipynb:506]: Relative linking (https://www.tensorflow.org/community/contribute/docs_style#links_in_markdown_and_notebooks)

Let's try the following:

```suggestion
        "`tf.distribute.Strategy` is integrated into `tf.keras`, which is TensorFlow's implementation of the [Keras API specification](https://keras.io/api/). `tf.keras` is a high-level API to build and train models. By integrating into the `tf.keras` backend, it's seamless for you to distribute your training written in the Keras training framework [using Model.fit](./keras/customizing_what_happens_in_fit.ipynb).\n",
```

File: site/en/guide/distributed_training.ipynb [hunk 2]
```diff
@@ -503,14 +503,14 @@
...
         "\n",
         "Here's what you need to change in your code:\n",
         "\n",
         "1. Create an instance of the appropriate `tf.distribute.Strategy`.\n",
         "2. Move the creation of Keras model, optimizer and metrics inside `strategy.scope`.\n",
         "\n",
-        "TensorFlow distribution strategies support all types of Keras models—[Sequential](/keras/sequential_model.ipynb), [Functional](/keras/functional.ipynb), and [subclassed](/keras/custom_layers_and_models.ipynb).\n",
+        "TensorFlow distribution strategies support all types of Keras models—[Sequential](/keras-io/guides/ipynb/sequential_model.ipynb), [Functional](keras-io/guides/ipynb/functional_api.ipynb), and [subclassed](keras-io/guides/ipynb/making_new_layers_and_models_via_subclassing.ipynb).\n",
```
Comment (8bitmp3) [inline site/en/guide/distributed_training.ipynb:513]: Relative linking (https://www.tensorflow.org/community/contribute/docs_style#links_in_markdown_and_notebooks)

Let's try the following:

```suggestion
        "TensorFlow distribution strategies support all types of Keras models—[Sequential](./keras/sequential_model.ipynb), [Functional](./keras/functional.ipynb), and [subclassed](./keras/custom_layers_and_models.ipynb)\n",
```

File: site/en/guide/distributed_training.ipynb [hunk 3]
```diff
@@ -636,7 +636,7 @@
       "source": [
         "## Use tf.distribute.Strategy with custom training loops\n",
         "\n",
-        "As demonstrated above, using `tf.distribute.Strategy` with Keras `Model.fit` requires changing only a couple lines of your code. With a little more effort, you can also use `tf.distribute.Strategy` [with custom training loops](/keras/writing_a_training_loop_from_scratch.ipynb).\n",
+        "As demonstrated above, using `tf.distribute.Strategy` with Keras `Model.fit` requires changing only a couple lines of your code. With a little more effort, you can also use `tf.distribute.Strategy` [with custom training loops](keras-io/guides/ipynb/writing_a_training_loop_from_scratch.ipynb).\n",
```
Comment (8bitmp3) [inline site/en/guide/distributed_training.ipynb:639]: Relative linking (https://www.tensorflow.org/community/contribute/docs_style#links_in_markdown_and_notebooks)

Let's try the following:

```suggestion
        "As demonstrated above, using `tf.distribute.Strategy` with Keras `Model.fit` requires changing only a couple lines of your code. With a little more effort, you can also use `tf.distribute.Strategy` [with custom training loops](./keras/writing_a_training_loop_from_scratch.ipynb).\n",
```

**Review by 8bitmp3 — APPROVED** [review]

File: site/en/guide/distributed_training.ipynb [hunk 4]
```diff
@@ -503,14 +503,14 @@
       "source": [
         "## Use tf.distribute.Strategy with Keras Model.fit\n",
         "\n",
-        "`tf.distribute.Strategy` is integrated into `tf.keras`, which is TensorFlow's implementation of the [Keras API specification](https://keras.io). `tf.keras` is a high-level API to build and train models. By integrating into the `tf.keras` backend, it's seamless for you to distribute your training written in the Keras training framework [using Model.fit](/keras/customizing_what_happens_in_fit.ipynb).\n",
+        "`tf.distribute.Strategy` is integrated into `tf.keras`, which is TensorFlow's implementation of the [Keras API specification](https://keras.io/api/). `tf.keras` is a high-level API to build and train models. By integrating into the `tf.keras` backend, it's seamless for you to distribute your training written in the Keras training framework [using Model.fit](./keras/customizing_what_happens_in_fit.ipynb).\n",
```
Comment (8bitmp3) [inline site/en/guide/distributed_training.ipynb:506]: Keras guides under `/guide` should have full links, they are hosted on keras.io. Fixed this:

```suggestion
        "`tf.distribute.Strategy` is integrated into `tf.keras`, which is TensorFlow's implementation of the [Keras API specification](https://keras.io/api/). `tf.keras` is a high-level API to build and train models. By integrating into the `tf.keras` backend, it's seamless for you to distribute your training written in the Keras training framework [using Model.fit](https://www.tensorflow.org/guide/keras/customizing_what_happens_in_fit).\n",
```

File: site/en/guide/distributed_training.ipynb [hunk 5]
```diff
@@ -636,7 +636,7 @@
       "source": [
         "## Use tf.distribute.Strategy with custom training loops\n",
         "\n",
-        "As demonstrated above, using `tf.distribute.Strategy` with Keras `Model.fit` requires changing only a couple lines of your code. With a little more effort, you can also use `tf.distribute.Strategy` [with custom training loops](/keras/writing_a_training_loop_from_scratch.ipynb).\n",
+        "As demonstrated above, using `tf.distribute.Strategy` with Keras `Model.fit` requires changing only a couple lines of your code. With a little more effort, you can also use `tf.distribute.Strategy` [with custom training loops](./keras/writing_a_training_loop_from_scratch.ipynb).\n",
```
Comment (8bitmp3) [inline site/en/guide/distributed_training.ipynb:639]: Keras guides under `/guide` should have full links, they are hosted on keras.io. Fixed this:

```suggestion
        "As demonstrated above, using `tf.distribute.Strategy` with Keras `Model.fit` requires changing only a couple lines of your code. With a little more effort, you can also use `tf.distribute.Strategy` [with custom training loops](https://www.tensorflow.org/guide/keras/writing_a_training_loop_from_scratch).\n",
```

**Review by 8bitmp3 — APPROVED** [review]

**Comment by 8bitmp3** [comment]
Thanks @MarkDaoust !

**Pull request merged**